### PR TITLE
Fix SpinBox.

### PR DESCRIPTION
**Pull request by vladest**
Fix variouse issues with spinbox
f.ex: https://highfidelity.manuscript.com/f/cases/14356/Developer-Network-Networking-Listening-Port-is-broken

Test plan:
- Open settings page with SpinBox, f.ex Avatar settings
- Edit values in spin boxes with different manner (enter from keyboard, using up/down buttons, slider)
- Make sure value is changed, saved and fits to its boudaries

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/8950/app_12943_PR_663af09605c57c7ebf7994879b1c3972c138d599.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8950/HighFidelity-Beta-PR12943-663af09605c57c7ebf7994879b1c3972c138d599.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8950/HighFidelity-Beta-Interface-PR12943-663af09605c57c7ebf7994879b1c3972c138d599.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8985/HighFidelity-Beta-PR12943-0e28e41107bac8670c020c41d28dbe1a806b8412.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/8985/HighFidelity-Beta-Interface-PR12943-0e28e41107bac8670c020c41d28dbe1a806b8412.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8985/HighFidelity-Beta-PR12943-0e28e41107bac8670c020c41d28dbe1a806b8412.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/8985/HighFidelity-Beta-Interface-PR12943-0e28e41107bac8670c020c41d28dbe1a806b8412.exe

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9019/app_12943_PR_60115752ad3fc9be0c3ae9cb0ad6801baf1f3563.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9019/HighFidelity-Beta-PR12943-60115752ad3fc9be0c3ae9cb0ad6801baf1f3563.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9019/HighFidelity-Beta-Interface-PR12943-60115752ad3fc9be0c3ae9cb0ad6801baf1f3563.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9019/HighFidelity-Beta-PR12943-60115752ad3fc9be0c3ae9cb0ad6801baf1f3563.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9019/HighFidelity-Beta-Interface-PR12943-60115752ad3fc9be0c3ae9cb0ad6801baf1f3563.exe

**Comment by sethalves** [comment]
https://highfidelity.fogbugz.com/f/cases/14349/Avatar-Settings-for-Scale-and-User-Height-are-displayed-and-saved-incorrectly

File: interface/src/ui/PreferencesDialog.cpp [hunk 1]
```diff
@@ -298,7 +298,7 @@ void setupPreferences() {
         {
             static const int MIN_PORT_NUMBER { 0 };
             static const int MAX_PORT_NUMBER { 65535 };
-            auto getter = [nodelist] { return static_cast<int>(nodelist->getSocketLocalPort()); };
+            auto getter = [nodelist] { qWarning() << "vladest: port" << static_cast<int>(nodelist->getSocketLocalPort()); return static_cast<int>(nodelist->getSocketLocalPort()); };
```
Comment (druiz17) [inline interface/src/ui/PreferencesDialog.cpp]: Can you remove this debug statement.

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9024/app_12943_PR_cf070e6e92c7b954f92694306567f87ac4d51e76.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9024/HighFidelity-Beta-PR12943-cf070e6e92c7b954f92694306567f87ac4d51e76.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/9024/HighFidelity-Beta-Interface-PR12943-cf070e6e92c7b954f92694306567f87ac4d51e76.dmg

**Comment by druiz17** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/9025/app_12943_PR_cf070e6e92c7b954f92694306567f87ac4d51e76.apk

**Comment by RyanDowne** [comment]
does this need to be in 67? Is it part of the interstitial work?

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9025/HighFidelity-Beta-PR12943-cf070e6e92c7b954f92694306567f87ac4d51e76.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/9025/HighFidelity-Beta-Interface-PR12943-cf070e6e92c7b954f92694306567f87ac4d51e76.exe

**Comment by qlimnavarro** [comment]
Currently Testing

**Comment by qlimnavarro** [comment]
Pass @conklin94122 
UI elements look and respond well in all views and platforms.

**Pull request merged**